### PR TITLE
fix(3000): Improve opacity of profile pictures

### PR DESCRIPTION
**Pull request by Twixes**
## Problem

This looked wrong:

## Changes

This looks right:

(Though, because of the background being transparent if there's a gravatar, there is now a flash of purple on app load.)

### CODE REVIEW

File: frontend/src/lib/lemon-ui/LemonButton/LemonButton3000.scss
Comment (Twixes) [inline frontend/src/lib/lemon-ui/LemonButton/LemonButton3000.scss:44]: This is generalized, so applies not just to ProfilePicture, but also e.g. Lettermark

**Comment by daibhin** [comment]
There's a lot of button changes that need to be merged in this PR: https://github.com/PostHog/posthog/pull/18949. Worried we'll have unnecessary merge conflicts if we make these changes on the old style buttons. I'm happy to work these changes into the new PR or hold off until after it has merged and do them as a follow up

**Review by benjackwhite — APPROVED** [review]

**Comment by Twixes** [comment]
Re-requesting your review @daibhin since I want to be sure I did styles correctly in the new buttons!

File: frontend/src/lib/lemon-ui/LemonButton/LemonButton3000.scss
Comment (daibhin) [inline frontend/src/lib/lemon-ui/LemonButton/LemonButton3000.scss:274]: Maybe beyond the scope of this PR but feels like this value could drift from https://github.com/PostHog/posthog/pull/19004/files#diff-c694fbbe6b6ff7273c964743f37751e5416009c150d0a9ddad1b4c031ef006d3R264. Perhaps the height should be a `calc` based on the button height minus some vertical padding

File: frontend/src/lib/lemon-ui/LemonButton/LemonButton3000.scss
Comment (daibhin) [inline frontend/src/lib/lemon-ui/LemonButton/LemonButton3000.scss:318]: Does `--lemon-button-icon-opacity: 0.5` work here?

File: frontend/__snapshots__/scenes-app-sidepanels--side-panel-settings.png
Comment (daibhin) [inline frontend/__snapshots__/scenes-app-sidepanels--side-panel-settings.png:1]: Hmm thought I fixed this in https://github.com/PostHog/posthog/pull/19045

File: frontend/__snapshots__/scenes-other-onboarding--onboarding-other-products.png
Comment (daibhin) [inline frontend/__snapshots__/scenes-other-onboarding--onboarding-other-products.png:1]: I've seen this flapping on a bunch of PRs now

**Review by daibhin — APPROVED** [review]

**Pull request merged**
